source/substitutions.txt

```diff
@@ -33,13 +33,16 @@
 
 .. |TGL| replace:: Tiger Lake
 
-.. |DDR4| replace:: Tiger Lake UP3 DDR4
```

Comment:
Difference shows these are removed, is that correct? if yes, why do we have to remove these?

Reply:
Yes this is intentional. I believe DDR4 should not be used as a substitution for TGL UP3 DD4 board. Hence I expanded these where used in source/supported-hardware/tgl-rvp.rst